ion-schema/src/schema.rs

```diff
@@ -53,15 +53,29 @@ impl Schema {
         SchemaTypeIterator::new(Arc::clone(&self.types), self.types.get_imports())
     }
 
+    /// Returns the requested type, if imported in this schema;
+    /// otherwise returns None.
```

Comment:
Documentation should mention that this includes built-in types.

Reply:
I think this should only include the imported types for built-in types there is already `get_local_or_imported_type` API. This is just to keep the imports separate. I have modified the logic for it in the `TypeStore`.